JSDemos/Demos/Scheduler/CustomizeIndividualViews/description.md

```diff
@@ -1 +1,3 @@
-In&nbsp;this demo, the _&laquo;Week&raquo;_ and _&laquo;Work Week&raquo;_ views of&nbsp;the Scheduler component are configured individually. The appointments are grouped by&nbsp;type (on&nbsp;the _&laquo;Week&raquo;_ view) and by&nbsp;priority (on&nbsp;the _&laquo;Work Week&raquo;_ view). In&nbsp;addition, the time window on&nbsp;the _&laquo;Work Week&raquo;_ view is&nbsp;narrowed down to&nbsp;working hours.
\ No newline at end of file
+To customize a view, configure its settings in an object inside the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. You should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties to override global view settings.
+
+This demo shows four views, two of which (Week and Work Week) are customized. Both customized views [group appointments](/Demos/WidgetsGallery/Demo/Scheduler/GroupOrientation/) by [resources](/Demos/WidgetsGallery/Demo/Scheduler/Resources/) and use the [dateCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#dateCellTemplate) to change the appearance of date cells. In addition, the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) properties are used to narrow down the time window on the Work Week view.
```

Comment:
`... two of which ...` - sign of a complicated sentence. You can even put a period after `four views` and start the next sentence with `Week and Work Week views`. 

`are customized` - passive. 

'properties are used' - passive. 

The list of changes is written as a narrative, where I would suggest a more structured approach (a list). 

So here's the suggestion that fixes all those comments. 

```suggestion
This demo customizes two views - Week and Work Week - in the following manner: 

* Both views [group appointments](/Demos/WidgetsGallery/Demo/Scheduler/GroupOrientation/) by [resources](/Demos/WidgetsGallery/Demo/Scheduler/Resources/).
* Both views use the [dateCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#dateCellTemplate) to change the appearance of date cells.
* The Work Week view changes the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) properties to limit the time range.

Day and Month views use default settings.
```